### PR TITLE
1letter/svg flag support

### PR DESCRIPTION
**Pull request by 1letter**
use SVG Flags in Language Selector

### Related PRs:
https://github.com/plone/plone.staticresources/pull/143
https://github.com/plone/plone.i18n/pull/36
https://github.com/plone/plone.app.i18n/pull/8
https://github.com/plone/plone.app.multilingual/pull/386
upcoming PR in buildout.coredev for old 5.x branches

### CODE REVIEW

File: src/plone/staticresources/static/components/svg-country-flags/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+country-flags
+=============
+
+You can view all the flags here: http://hjnilsson.github.io/country-flags/
```
Comment (thet) [inline src/plone/staticresources/static/components/svg-country-flags/README.md:4]: This website does not exist. It's prpbably this one: https://hampusborgos.github.io/country-flags/
Please change.

**Review by thet — CHANGES_REQUESTED** [review]
Please give me a hint why we need country flags in Plone. If this is for language switching, country flags are a no-go. Which country would you use for "German", "English", "Spanish" language? See: http://www.flagsarenotlanguages.com/blog/best-practice-for-presenting-languages/
If we have the option to show flags for language switching, I rather suggest submitting a PLIP to remove it.

But maybe there is another use case.

Before going on, you need to convince me that we need this at all.

To the PR itself:

- Why do we need png files? AFAIK we don't include png files for bootstrap icons and imply all browsers can support svg, which is actually safe nowadays. If the png files come with the npm package along with svg files, I'd add a .gitignore entry to https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/.gitignore
- Please try to reduce the amount of added files. I doubt we have nearly 1300+ different icons for the world's countries and languages. I dislike adding so much new assets to this package where in the es6 branch I remove almost all components from ``plone.staticresources.static.components``. I think there could be easily much less.

Also, please seperate your commits like so:

- 1 commit for the functionality itself - adding the scripts, adding dependencies to package.json, adding other manually created content.
- 1 commit for yarn.lock, npm.lock or similar files (because they are often the cause for merge conflicts and this way this single commit can simply be removed and re-created).
- 1 commit adding the assets and resources. This commit is the one no one will look at for checking what's going on, because it's too big.

File: news/140.breaking [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Use SVG Flags in Language Selector
```
Comment (jensens) [inline news/140.breaking:1]: I do not think the "use" is part of the change in this package here

Comment (jensens) [reply news/140.breaking:1]: Additional, this is not a breaking change for this package.

File: src/plone/staticresources/static/components/svg-country-flags/README.md (hunk 1 above)
Comment (jensens) [reply src/plone/staticresources/static/components/svg-country-flags/README.md:4]: probably this was copied from the original npm pkg. It was also fixed upstream already https://github.com/hampusborgos/country-flags/commit/2c5cc97ffdb444a830efb3da6f9ef4149d1220c5#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Comment (1letter) [reply src/plone/staticresources/static/components/svg-country-flags/README.md:4]: @thet thanks for the hints. I will take it into account.

Comment (thet) [reply src/plone/staticresources/static/components/svg-country-flags/README.md:4]: Ah this comes from the external package. Didn't see that.

**Review by thet — APPROVED** [review]

**Comment by thet** [comment]
@1letter tnx for the changes. I just approved the PR.

**Pull request merged**